### PR TITLE
feat(test): improve tests coverage

### PR DESCRIPTION
**Pull request by CarlChaoCarl**
**What does this PR do?**
improve tests coverage

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/build.gradle [hunk 1]
```diff
@@ -2,12 +2,14 @@ plugins {
     id "org.gradle.test-retry" version "1.5.9"
     id "org.sonarqube" version "2.6"
     id "com.gorylenko.gradle-git-properties" version "2.4.1"
+    id "io.github.surpsg.offlins" version "0.3.0"
```
Comment (lxcmyf) [inline framework/build.gradle:5]: What is the purpose of introducing this dependency?

File: framework/build.gradle [hunk 2]
```diff
@@ -121,10 +123,12 @@ test {
     testLogging {
         exceptionFormat = 'full'
     }
+    /*
```
Comment (lxcmyf) [inline framework/build.gradle:126]: Why annotate this？

File: framework/build.gradle (hunk 1 above)
Comment (chaosuper123321) [reply framework/build.gradle:5]: To solve the incompatibility problem between jacoco and powermock

File: framework/build.gradle (hunk 2 above)
Comment (chaosuper123321) [reply framework/build.gradle:126]: Use the report part configuration of the offlinsCoverage module

File: framework/build.gradle [hunk 3]
```diff
@@ -149,6 +154,26 @@ jacocoTestReport {
...
+        xml.location.set project.file('build/reports/jacoco/test/jacocoTestReport.xml')
+        // Optional. By default `build/reports/jacoco/coverageReport.xml`
+
+        csv.enabled.set true // Optional. By default `false`
+        csv.location.set project.file('build/reports/jacoco/test/csvCoverage.csv')
+        // Optional. By default `build/reports/jacoco/coverageReport.csv`
+    }
+}
+
+
```
[the remark this answers is not in the document]
Comment (chaosuper123321) [reply framework/build.gradle:176]: The default output path configured by offlinsCoverage is inconsistent with the original configuration of jacoco. In order to maintain the consistency of the coverage report document path.

File: framework/src/test/java/org/tron/common/logsfilter/TransactionLogTriggerCapsuleMockTest.java [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+    TransactionLogTriggerCapsule.class,
+    TransactionTrace.class
+})
+public class TransactionLogTriggerCapsuleMockTest {
+
+  private static final String OWNER_ADDRESS = "41548794500882809695a8a687866e76d4271a1abc";
+  private static final String RECEIVER_ADDRESS = "41abd4b9367799eaa3197fecb144eb71de1e049150";
+  private static final String CONTRACT_ADDRESS = "111111";
```
[the remark this answers is not in the document]
Comment (chaosuper123321) [reply framework/src/test/java/org/tron/common/logsfilter/TransactionLogTriggerCapsuleMockTest.java:44]: OK, got it

File: framework/build.gradle (hunk 3 above)
Comment (lxcmyf) [inline framework/build.gradle:176]: Can the Jacoco coverage of each module still be output locally after this change?

Comment (chaosuper123321) [reply framework/build.gradle:176]: Yes, and the path is still the same

Comment (chaosuper123321) [reply framework/build.gradle:176]: Yes, and the path is still the same

File: framework/build.gradle (hunk 1 above)
Comment (CarlChaoCarl) [reply framework/build.gradle:5]: To solve the incompatibility problem between jacoco and powermock

File: framework/build.gradle (hunk 2 above)
Comment (CarlChaoCarl) [reply framework/build.gradle:126]: Use the report part configuration of the offlinsCoverage module

File: framework/build.gradle (hunk 3 above)
[the remark this answers is not in the document]
Comment (CarlChaoCarl) [reply framework/build.gradle:176]: The default output path configured by offlinsCoverage is inconsistent with the original configuration of jacoco. In order to maintain the consistency of the coverage report document path.

File: framework/src/test/java/org/tron/common/logsfilter/TransactionLogTriggerCapsuleMockTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (CarlChaoCarl) [reply framework/src/test/java/org/tron/common/logsfilter/TransactionLogTriggerCapsuleMockTest.java:44]: OK, got it.

File: framework/build.gradle (hunk 3 above)
Comment (CarlChaoCarl) [reply framework/build.gradle:176]: Yes, and the path is still the same

**Review by lvs007 — APPROVED** [review]

**Review by lxcmyf — APPROVED** [review]

**Pull request merged**